dockers/skylab/bwa/README.md

```diff
@@ -0,0 +1,35 @@
+# BWA
+
+## Quick reference
+
+Copy and paste to pull this image
+
+#### `docker pull us.gcr.io/broad-gotc-prod/bwa:1.0.0-0.7.17-1660770463`
+
+
+- __What is this image:__ This image is an Ubuntu-based custom image for running BWA.
+- __What is BWA:__ BWA is a software package for mapping DNA sequewnces against a large reference genome, [more info](http://bio-bwa.sourceforge.net).
```

Comment:
```suggestion
- __What is BWA:__ BWA is a software package for mapping DNA sequences against a large reference genome, [more info](http://bio-bwa.sourceforge.net).
```

Reply:
I definitely corrected that in the other one that I copied! Thanks for flagging